plugins/catalog-backend/src/modules/github/GithubOrgReaderProcessor.ts

```diff
@@ -67,12 +69,14 @@ export class GithubOrgReaderProcessor implements CatalogProcessor {
     integrations: ScmIntegrationRegistry;
     logger: Logger;
     githubCredentialsProvider?: GithubCredentialsProvider;
+    useOrgAsNamespace?: Boolean;
```

Comment:
Wondering if maybe having the ability to control the namespace would be more useful then a flag? Possible support a map of orgs to namespaces along with an auto option that works like the flag.

Reply:
I think the difference with this processor is that you can specify each org as a location rather than listing the orgs in the config and I can register and unregister them at runtime. Is that right? This one seems to be more flexible.

I believe the code owners file uses the same namespacing as this patch is allowing. i.e.`@<org>/<team>` would come in as `<org>/<team>` rather than `default/<team>`.

In anycase, as I understand it, I have made this feature optional so the default case pertains unchanged.